gluster_bridge.spec

```diff
@@ -0,0 +1,75 @@
+%define pkg_name tendrl-gluster-bridge
+%define pkg_version 0.0.1
+%define pkg_release 1
+
+Name: %{pkg_name}
+Version: %{pkg_version}
+Release: %{pkg_release}%{?dist}
+BuildArch: noarch
+Summary: Common Module for All Bridges
+Source0: %{pkg_name}-%{pkg_version}.tar.gz
+Group:   Applications/System
+License: GPLv2+
+Url: https://github.com/Tendrl/gluster_bridge
+
+#Requires: python-etcd
```

Comment:
By mistakenly i have commented that line during the test build. I will fix this.